src/components/Graphs/Legend.module.css

```diff
@@ -0,0 +1,32 @@
+.legendContainer {
+  padding: 1rem;
+  min-width: 240px;
+  max-width: 300px;
+  max-height: 280px;
+  border: 1px solid #ccc;
+  border-radius: 8px;
+  background-color: #fff;
+  margin: 1rem;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.05);
+  overflow: auto;
+  align-self: flex-start;
+}
+
+.legendTitle {
+  margin-bottom: 10px;
+}
+
+.legendRow {
+  display: flex;
+  align-items: center;
+  margin-bottom: 8px;
+}
+
+.legendColorBox {
+  width: 16px;
+  height: 16px;
+  background-color: var(--legend-color, #ccc);
```

Comment:
Is `--legend-color` defined anywhere? Could as well just leave it out, since it’s overridden by JS anyway